docset/windows/international/Set-Culture.md

```diff
@@ -69,7 +69,7 @@ This cmdlet supports the common parameters: -Debug, -ErrorAction, -ErrorVariable
 ## OUTPUTS
 
 ## NOTES
-
+In order for changes made by this cmdlet to take effect you need to open a new PowerShell session.
```

Comment:
What about
_Changes made by the use of this cmdlet will take effect on subsequent PowerShell sessions._

Reply:
Sounds way better :) I will update that.